service-commands/src/commands/service-commands.json

```diff
@@ -66,7 +66,7 @@
  "solana-programs": {
     "path": "solana-programs",
     "up": [
-      "cd solana-programs/; docker run --network audius_dev -e SOLANA_HOST='http://solana:8899' -v \"$PWD:/usr/src/app\" audius/solana-programs:develop > solana-program-config.json 2>$PROTOCOL_DIR/service-commands/output.log",
+      "cd solana-programs/; docker run --network audius_dev -e SOLANA_HOST='http://solana:8899' -v \"$PWD:/usr/src/app\" audius/solana-programs:develop2 > solana-program-config.json 2>$PROTOCOL_DIR/service-commands/output.log",
```

Comment:
This is me pushing an image manually built on ubuntu host - if we want to overwrite the main program for now that is fine, but this should also work as expected

Reply:
* program container

Reply:
if we do this should we add a TODO / task to update this to overwrite the main?

Reply:
yes definitely